CRD-DTR/README.md

```diff
@@ -0,0 +1,86 @@
+# CDS-Library/CRD-DTR
```

Comment:
We probably need to add README for the rule sets under Example folder as well. Basically, in order for the example rule sets to work, they need to follow the similar file layout and naming conventions.